Library/Formula/m2c.rb

```diff
@@ -0,0 +1,17 @@
+class M2c < Formula
+  desc "Modula-2 to C compiler"
+  homepage "https://savannah.nongnu.org/projects/m2c/"
+  url "http://download.savannah.gnu.org/releases/m2c/0.7/m2c-0.7.tar.gz"
+  sha256 "b725ed617f376e1a321e059bf1985098e950965d5edab161c6b24526f10a59bc"
+
+  def install
+    system "./configure", "+cc=gcc"
```

Comment:
Use `ENV.cc` here instead of hardcoding GCC.  If it doesn't build with Clang, you can use `fails_with :clang` (see [`ape.rb`](https://github.com/Homebrew/homebrew/blob/1777fc04b5afbc1ed309361c1324cfe62ce32f62/Library/Formula/ape.rb#L16-L19) for an example)—but that should also be filed upstream.